tests/monitoring/monitoring.go

```diff
@@ -118,55 +118,31 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 	})
 
 	Context("System Alerts", func() {
-		disableVirtHandler := func() {
-			originalKv := libkubevirt.GetCurrentKv(virtClient)
-			customizedComponents := v1.CustomizeComponents{
-				Patches: []v1.CustomizeComponentsPatch{
-					{
-						ResourceName: virtHandler.deploymentName,
-						ResourceType: "DaemonSet",
-						Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
-						Type:         v1.StrategicMergePatchType,
-					},
-				},
-			}
-
-			patchBytes, err := patch.New(patch.WithAdd("/spec/customizeComponents", customizedComponents)).GeneratePayload()
-			Expect(err).ToNot(HaveOccurred())
-			_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
-			Expect(err).ToNot(HaveOccurred())
+		var originalKv *v1.KubeVirt
+		var virtHandlerName string
 
-			Eventually(func() string {
-				vh, err := virtClient.AppsV1().DaemonSets(originalKv.Namespace).Get(context.Background(), virtHandler.deploymentName, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				return vh.Spec.Template.Spec.NodeSelector[k8sv1.LabelHostname]
-			}).WithTimeout(90 * time.Second).WithPolling(5 * time.Second).Should(Equal("does-not-exist"))
-
-			Eventually(func() int {
-				vh, err := virtClient.AppsV1().DaemonSets(originalKv.Namespace).Get(context.Background(), virtHandler.deploymentName, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				return int(vh.Status.NumberAvailable + vh.Status.NumberUnavailable)
-			}).WithTimeout(90 * time.Second).WithPolling(5 * time.Second).Should(Equal(0))
-		}
-
-		restoreVirtHandler := func() {
-			originalKv := libkubevirt.GetCurrentKv(virtClient)
-			patchBytes, err := patch.New(patch.WithRemove("/spec/customizeComponents")).GeneratePayload()
-			Expect(err).ToNot(HaveOccurred())
-
-			_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
-			Expect(err).ToNot(HaveOccurred())
-		}
+		BeforeEach(func() {
+			originalKv = libkubevirt.GetCurrentKv(virtClient)
+			virtHandlerName = virtHandler.deploymentName
```

Comment:
why do we need this?
Can't we drop it in favor of using `virtHandler.deploymentName`?

Reply:
we can, but if it will change in the future we can change only the var instead of changing each occurrences.
WDYT?

Reply:
In case of future change, I'd expect it to be changed here :)
https://github.com/kubevirt/kubevirt/blob/6ffe6c1b701ae76727c84742cbbe92cb37cd004f/tests/monitoring/component_monitoring.go#L74
No?

Reply:
@fossedihelm fixed (: